invenio_previewer/webpack.py

```diff
@@ -76,14 +76,16 @@
                 'simple_image_css':
                     './scss/invenio_previewer/simple_image.scss',
                 'txt_css': './scss/invenio_previewer/txt.scss',
+                'nbconvert_index_css': './../../../share/jupyter/nbconvert/templates/lab/static/index.css',
```

Comment:
Please, check these two. I'm not sure if this is the proper way to include them

Reply:
I don't see these files in your PR, did you push them ?

Reply:
These files come from the dependencies of the package, from nbconvert. Is there a better way of including css from dependencies, instead of putting them in the `entry points`?

Reply:
see https://discord.com/channels/692989811736182844/1092367535141289984/1149642305645588530